.github/workflows/README.md

```diff
@@ -0,0 +1,39 @@
+# Decidim GitHub Actions workflows
+
+We use GitHub Actions as CI.
+
+- `lint_code.yml`: runs the linters for Ruby, JS and ERB files.
+- `ci_main.yml`: runs the tests for the main folder
+- `ci_core.yml`: runs the tests for the `decidim-core` module. The remaining workflows (except noted) are based on this one.
+
+Individual workflows with changes:
+
+- `ci_generators.yml`: Does not need to create the test app, so this command is removed. Screenshots uploads and chromedriver setup steps are also removed. We also customize the gems path after running `bundle install`:
```

Comment:
Please excuse my lack of knowledge related with github actions but why it is not needed to create the test_app? is it generated  somewhere else?

Reply:
I mean, it is generated in each module. Maybe you meant that decidim-generators does not need the test_app?
Probably... this is also why you mention that Screenshots and chromedriver are not needed

Reply:
The generators module test suite does not generate the test app in CircleCI, so it's not generated either in Github Actions.

You're correct for chromedriver and screenshots!